src/format.cc

```diff
@@ -125,7 +125,7 @@ void Format::RebuildSegments() {
     // which rounds up to a vec4.
     //
     // In std140 and std430 a vector of size 3N will round up to a vector of 4N.
```

Comment:
This is an independent issue, but I am a little bit confused. Based on this comment, it should check `if layout_ == Layout::kStd140 or layout_ == Layout::kStd430` ?

Reply:
Maybe when we have more layouts. Right now only 140 and 430 exist.